### ISSUE TITLE
upgrade addons-server dev and local to django2.2

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#11020]
We've fixed all the failing tests, and it appears to work fine, so time to put it on addons-dev to give it a proper test.  Stage and Prod to follow only when we're confident it's stable (and after the full django2.2 release in early April)

**Comment by AlexandraMoga** [comment]
@eviljeff last time QA verified a django upgrade, we followed the instructions mentioned in https://github.com/mozilla/addons-server/issues/5271#issuecomment-404046137 .
I'm guessing the same approach would work this time as well.

**Comment by eviljeff** [comment]
@AlexandraMoga yes.  Should be less involved this time as there's no legacy frontend (apart from /statistics).

**Comment by ioanarusiczki** [comment]
@eviljeff @AlexandraMoga 
I verified on AMO dev with FF66 based on the given [instructions](https://github.com/mozilla/addons-server/issues/5271#issuecomment-404046137) from the last django upgrade (to 1.1).
- login, register, upload of an add-on work as expected.
- translations and editing them into an add-on work alright.
- the last years dev hub regressions are not reproducible.
- I did a couple of basic testing in django admin -dev too, gropus, collections, editing discovery items seem to work just fine.
